### ISSUE TITLE
ARM Linux build

### ISSUE THREAD

**Issue by elyxlz** [cursor/cursor#1532]
**Is your feature request related to a problem? Please describe.**
I am on asahi linux and can't use cursor as it only has x64 intel builds.

**Describe the solution you'd like**
An ARM linux build.

**Comment by tohodakilla** [comment]
bump

**Comment by kylecarbs** [comment]
I just created some unofficial builds that work on ARM. Please try them out and let me know if they work for you!

https://github.com/coder/cursor-arm

**Comment by a0s** [comment]
@kylecarbs Does it helps to connect cursor from mac os to remote armhf host?

**Comment by tenfourty** [comment]
Seems like they have [official ARM builds now](https://github.com/getcursor/cursor/issues/1410):

> Cursor's official arm builds are now available at cursor.sh. 🥳
> 
> Though if you download directly, it will still download x86. You need to use these links:
> 
> Linux: https://downloader.cursor.sh/linux/appimage/arm64
> Windows: https://downloader.cursor.sh/windows/nsis/arm64

**Comment by zeeyado** [comment]
@tenfourty 
That's great, and I have got it working with both wayland and x11 in Android 15's new linux terminal (some finicky issues, but usable)

But while the desktop version is at 0.47.7, the arm64 appimage is at 0.45.14, with the old Composer, etc. 

Where did you find these links? Some else found this also: https://github.com/getcursor/cursor/issues/1410#issuecomment-2581526039

**Comment by tenfourty** [comment]
> [@tenfourty](https://github.com/tenfourty) That's great, and I have got it working with both wayland and x11 in Android 15's new linux terminal (some finicky issues, but usable)
> 
> But while the desktop version is at 0.47.8, the arm64 appimage is at 0.45.14, with the old Composer, etc.

Indeed, I can see that as well, I'm not sure why their build is not up to date!

> 
> Where did you find these links? Some else found this also: [#1410 (comment)](https://github.com/getcursor/cursor/issues/1410#issuecomment-2581526039)

Yes, this is where I found the links: https://github.com/getcursor/cursor/issues/1410#issuecomment-2569279736
